doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -91,8 +91,6 @@ Developing with Front-End Modules
 
 * Added support for the following:
 
-  * :ref:`nRF2220 Front-End Module <ug_radio_fem_nrf2220>`.
-  * :ref:`nRF2220 EK shield <ug_radio_fem_nrf2220ek>`.
```

Comment:
```suggestion
  * nRF2220 Front-End Module.
  * nRF2220 EK shield.
```
In old release notes, we normally don't remove the whole ref label entry. Just keep as a plain text. 
Applies to other release notes as well.